spec/lib/tasks/backfill_idv_level_rake_spec.rb

```diff
@@ -43,6 +43,11 @@
     expect(profiles[:unsupervised_no_level].idv_level).to be_nil
     expect(profiles[:in_person].idv_level).not_to be_nil
     expect(profiles[:in_person_no_level].idv_level).to be_nil
+
+    # NULL values can blow up postgres WHERE NOT IN() conditions if you're
+    # not careful...
+    create(:in_person_enrollment, profile: nil)
```

Comment:
also these profiles are likely from some bug right?

Reply:
I don't think so, because they get created early in the IDV flow, before the profile is minted. (see https://github.com/18F/identity-idp/blob/main/app/controllers/idv/in_person/usps_locations_controller.rb#L89-L95)